From e036b5e15613d66c3e31faac82509ee09d81654c Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Mon, 24 Dec 2018 11:25:23 +0800
Subject: [PATCH 441/706] ptp: add ptp_qoriq support for ENETC

There is same 1588 timer IP on ENETC Ethernet controller.
So added FSL_ENETC as a dependency of ptp_qoriq driver
so that functions in ptp_qoriq driver could be reused by
ENETC PTP driver which is a PCI driver.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
(cherry picked from commit b34e6dc3f82a59827a78b736986b4fe394f73c3f)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/ptp/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ptp/Kconfig b/drivers/ptp/Kconfig
index d137c480db46..7c5903149cf0 100644
--- a/drivers/ptp/Kconfig
+++ b/drivers/ptp/Kconfig
@@ -43,7 +43,7 @@ config PTP_1588_CLOCK_DTE
 
 config PTP_1588_CLOCK_QORIQ
 	tristate "Freescale QorIQ 1588 timer as PTP clock"
-	depends on GIANFAR || FSL_DPAA_ETH
+	depends on GIANFAR || FSL_DPAA_ETH || FSL_ENETC
 	depends on PTP_1588_CLOCK
 	default y
 	help
-- 
2.17.1

